index.js

```diff
@@ -10,17 +10,17 @@ const {
   ScrollView,
   Platform,
   StyleSheet,
-  ViewPagerAndroid,
   InteractionManager,
 } = ReactNative;
 const TimerMixin = require('react-timer-mixin');
+import ViewPager from "@react-native-community/viewpager";
```

Comment:
Please use **require**, **import** could return`undefined` in projects with Typescript